From 1d39cfca2ca3a5026eea26eb91e4b7bf2bb77331 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 20 Aug 2018 14:28:48 +0800
Subject: [PATCH] ANDROID: ARM: setup: Increase COMMAND_LINE_SIZE to 4096 bytes

With features AVB / dm-verity enabled, cmdline content is about to
exceed previous maximum 2048 bytes. Increase the cmdline buffer size
(COMMAND_LINE_SIZE) to 4096 bytes for other features to use.

Change-Id: I73baf92e5dce1a341d523b1a976d387e794b3998
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/include/uapi/asm/setup.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/include/uapi/asm/setup.h b/arch/arm/include/uapi/asm/setup.h
index b4d1bea21a2d..ebe54c5963ae 100644
--- a/arch/arm/include/uapi/asm/setup.h
+++ b/arch/arm/include/uapi/asm/setup.h
@@ -16,7 +16,7 @@
 
 #include <linux/types.h>
 
-#define COMMAND_LINE_SIZE 2048
+#define COMMAND_LINE_SIZE 4096
 
 /* The list ends with an ATAG_NONE node. */
 #define ATAG_NONE	0x00000000
-- 
2.35.3

